lib/lumberjack/server.rb

```diff
@@ -214,13 +226,14 @@ def initialize(fd)
       super()
       @parser = Parser.new
       @fd = fd
+      @closed = Concurrent::AtomicBoolean.new
```

Comment:
Ahh, I see it now. THanks! I think it is OK to close explicitly in the plugin, doesn't have to be in the library here.

Can you add a comment to the `close` method to indicate that it is up to the caller to close each connection explicitly? I'm OK with the document-it approach for this.